common/views/components/RelevanceRater/RelevanceRater.js

```diff
@@ -9,34 +9,60 @@ type Props = {|
   query: string,
   page: ?number,
   workType: ?(string[]),
+  productionDatesTo: ?string,
+  productionDatesFrom: ?string,
+  sort: ?string,
+  sortOrder: ?string,
   _queryType: ?string,
 |};
 
+function storeRating(key, value) {
+  window.localStorage.setItem(key, value.toString());
+}
+
+function getRating(key) {
+  return window.localStorage.getItem(key)
+    ? parseInt(window.localStorage.getItem(key))
+    : 0;
+}
+
 const RelevanceRater = ({
   id,
   position,
   query,
   page,
   workType,
+  productionDatesTo,
+  productionDatesFrom,
+  sort,
+  sortOrder,
   _queryType,
 }: Props) => {
   const [isEnhanced, setIsEnhanced] = useState(false);
+  const [currentlyRatedValue, setCurrentlyRatedValue] = useState(0);
+  const trackingObject = {
+    id,
+    position,
+    query,
+    page: page || 1,
+    workType,
+    productionDatesTo,
+    productionDatesFrom,
+    sort,
+    sortOrder,
+    _queryType,
+  };
```

Comment:
[See here](https://github.com/wellcometrust/wellcomecollection.org/pull/4863/files#r346777711)

Reply:
To be able to see this, take a look at the network request that goes through.